src/mlpack/methods/ann/rnn.hpp

```diff
@@ -70,6 +70,15 @@ class RNN
       OutputLayerType outputLayer = OutputLayerType(),
       InitializationRuleType initializeRule = InitializationRuleType());
 
+  //! Copy constructor.
+  RNN(const RNN&);
+
+  //! Move constructor.
+  RNN(RNN&&);
+
+  //! Copy/move assignment operator.
+  RNN& operator = (RNN);
```

Comment:
Maybe I overlooked it, but I don't see an implementation of this function.

Reply:
Was trying to follow the convention used in ```mlpack/methods/ann/ffn.hpp``` but now have changed it to be clearer. Both the copy and move implementations are in ```rnn_impl.hpp```.